eng/scripts/Get-BinarySizes.ps1

```diff
@@ -48,10 +48,11 @@ function getTargetOs {
         return $OsVMImage
     }
 
-    if ($OsVMImage -eq "MMS2019") {
+    if ($OsVMImage -match "^MMS2019$|^win-2019$") {
```

Comment:
We should try and work out the Windows version here using `$IsWindows` and something like `[environment]::OSVersion`

Reply:
Unfortunately `[environment]::OSVersion` doesn't give us much detail on the OS name that would be useful for what this code is trying to produce. Let's keep what you've got in place for now. I can clean this up in a later version.